setup.py

```diff
@@ -18,6 +18,7 @@
     url='https://github.com/kingthomasc/python-purify',
     download_url='https://github.com/kingthomasc/python-purify/tarball/{ver}'.format(
         ver=version_num),
+    install_requires=['abc', 'six', 'xml'],
```

Comment:
Let me know your thoughts on this.  I ran into an issue when using a brand new ec2 image - `six` was missing and I had to add it manually.  Not sure if `abc` and `xml` are included as part of every python distribution.